docs: fix Spanish documentation typos

miguelpalazzo opened the pull request:
Hope you don't mind, I was reading the Spanish docs and found some typos on numerical values. In Spanish, thousands should be separated with a dot, instead of a comma, as it was displayed on docs.

Before proceeding to fix it, I've also found that the Open Source contributing guidelines were pointing to the English document on another repository, so I have made the fix for it to correctly point to its Spanish counterpart.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

miguelpalazzo committed:
Fix numerical representation on the Spanish docs

In the Spanish language, a dot should be used as thousands separator,
instead of a comma, like it happens in English.

This solves the situation for any numerical representation found over
the Spanish docs, to make it look professionally appealing for Spanish-
speaking audiences.

miguelpalazzo committed:
Fix link to Open Source contributing from Spanish docs

RandellDawson commented:
@miguelpalazzo Thank you for  creating this PR.  However, since it was created, the underlying document has changed.  Can you please review and resolve the current merge conflicts?  It could be the changes to the underlying document are so significant that your changes are no longer relevant.  If the latter is the case, it might be easier to just close this PR and create a new PR with any changes you still feel are important to make.

If you need any assistance or have questions about resolving the merge conflicts, please reply back or contact someone in the [Contributors Room on Gitter](https://gitter.im/FreeCodeCamp/Contributors).

Thank you

miguelpalazzo committed:
Merge branch 'master' into spanish-fixes

miguelpalazzo committed:
Fix thousand separator for Spanish README

In Spanish, thousands separator must be represented with a dot, instead of a comma. Comma is used to separate decimal digits.

miguelpalazzo commented:
@RandellDawson please check again if everything it's correct :) Thank you!

RandellDawson commented:
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉
